docs/api/classifier/class_prediction_error.rst

```diff
@@ -19,13 +23,39 @@ The class prediction error chart provides a way to quickly understand how good y
     # Perform 80/20 training/test split
     X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.20,
                                                         random_state=42)
+    # Instantiate the classification model and visualizer
+    visualizer = ClassPredictionError(
+        RandomForestClassifier(), classes=classes
+    )
 
-.. code:: python
+    # Fit the training data to the visualizer
+    visualizer.fit(X_train, y_train)
 
-    from sklearn.ensemble import RandomForestClassifier
+    # Evaluate the model on the test data
+    visualizer.score(X_test, y_test)
 
+    # Draw visualization
+    visualizer.poof()
+
+
+.. plot::
+    :context: close-figs
+    :include-source: False
+    :nofigs:
+
+    from sklearn.model_selection import train_test_split
+    from sklearn.ensemble import RandomForestClassifier
     from yellowbrick.classifier import ClassPredictionError
+    from yellowbrick.datasets import load_credit
 
+    X, y = load_credit()
+    
+    classes = ['default', 'current']
```

Comment:
Just for interpretability, do you mind changing this to something like: 
```classes = ['account in default', 'current with bills']```
(assuming the text will fit :D )